app/components/MemberList/MemberList.tsx

```diff
@@ -0,0 +1,64 @@
+import { useMemberQuery } from '@camp/data-layer';
+import { FullPageLoader, showNotification } from '@camp/design';
+import { errorMessages, messages } from '@camp/messages';
+import { Center, Group, Stack, Title } from '@mantine/core';
+import { useState } from 'react';
+
+import { MemberEmptyState } from '../MemberEmptyState';
+import { CreateMemberButton, MemberForm } from '../MemberForm';
+
+interface Props {
+  familyId: string;
+}
+
+const t = messages.member;
+
+export const MemberList = ({ familyId }: Props) => {
+  const [memberForm, setMemberForm] = useState<React.ReactNode[]>([]);
+  const [isMemberEmpty, setIsMemberEmpty] = useState<boolean>();
+
+  const { data, error, loading } = useMemberQuery({
+    variables: { family_id: familyId },
+  });
+  const member = data?.members;
+
+  const addNewMemberHandler = () => {
+    setMemberForm(memberForm.concat(<MemberForm familyId={familyId} />));
+    setIsMemberEmpty(true);
+  };
+
```

Comment:
Seems good. But what's wrong with this way?

Reply:
I'm not really sure but I try to avoid storing complicated data inside a react state (like storing a function)
cause I think that might be a sign that smth is wrong.